### DISCUSSION TITLE
Avalonia Chinese Documentation 中文文档

### DISCUSSION THREAD

**Discussion by rabbitism** [AvaloniaUI/Avalonia#9485] [General]
Hello all, we have created a repository for hosting Avalonia Documentation in Simplified Chinese. 

[https://github.com/avaloniachina/Documentation](https://github.com/avaloniachina/Documentation)

Currently we only finished `Getting Started` part, but we will keep working on finishing and syncing all of the docs. 
You are welcomed to read the docs if you are comfortable with Chinese and welcomed if you would like to contribute to translation.

**Comment by robloo** [comment]
It's great to see the documentation being localized. However, shouldn't we have a common repository and infrastructure to support all localizations together?

**Reply by rabbitism** [reply]
You are correct, I raised this issue in Documentation https://github.com/AvaloniaUI/Documentation/issues/337 
But I think devs are working on V11 hard so let me host it somewhere else, and merge it back once infra is ready.

**Reply by robloo** [reply]
Ok, makes sense. Thanks for the background
